support-frontend/assets/components/subscriptionCheckouts/layout.module.scss

```diff
@@ -34,12 +34,32 @@
   .aside {
     position: sticky;
     top: 0;
-    z-index: 999;  
+    z-index: 99;
```

Comment:
The previous z-index was making asides appear above the consent banner.

Reply:
👍